pkg/doc/presexch/schema.go

```diff
@@ -230,3 +232,21 @@ const definitionSchema = `
         }
     }
 }`
+
+// PresentationSubmissionJSONLDContext based con this commit:
+// https://github.com/decentralized-identity/presentation-exchange/commit/980ee781976fa0a6a247807e879f7d7d0e7e8e0a.
+const PresentationSubmissionJSONLDContext = `{
```

Comment:
we already have it here `https://github.com/hyperledger/aries-framework-go/blob/809d27ce5111714a1a23c62648cec9d19fae13bd/pkg/doc/presexch/definition.go#L75`

Reply:
oh :)

Reply:
Substituted `presentationSubmissionContext` for the new exposed one.

Also substituted use of `presentationSubmissionContextURI` with the one already exposed.